includes/definitions/hatteras.yaml

```diff
@@ -0,0 +1,59 @@
+os: hatteras
+text: 'Hatteras Overdue DSLAM'
+type: network 
+icon: hatteras
+
+empty_ifdescr: true
+ifname: true
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    - sysDescr:
+            - HN4000-I
+            - HN4000e
+bad_if:
+  - notPresent
+  - serConsole
+good_if:
+  - ras #all interfaces has 'Hatteras' in it...
+discovery_modules:
```

Comment:
We prefer to not set these at all if not for good reason, like if the device crashes or so